zebra-chain/src/transaction/unmined.rs

```diff
@@ -91,23 +96,23 @@ impl UnminedTxId {
     /// [`Hash`] does not uniquely identify unmined v5 transactions.
     #[allow(dead_code)]
     pub fn from_legacy_id(legacy_tx_id: Hash) -> UnminedTxId {
-        Narrow(legacy_tx_id)
+        Legacy(legacy_tx_id)
     }
 
-    /// Return the unique ID for this transaction's effects.
+    /// Return the unique ID that will be used if this transaction gets mined into a block.
     ///
     /// # Correctness
     ///
-    /// This method returns an ID which uniquely identifies
-    /// the effects (spends and outputs) and
-    /// authorizing data (signatures, proofs, and scripts) for v1-v4 transactions.
+    /// For v1-v4 transactions, this method returns an ID which changes
+    /// if this transaction's effects (spends and outputs) change, or
+    /// if its authorizing data changes (signatures, proofs, and scripts).
     ///
-    /// But for v5 transactions, this ID only identifies the transaction's effects.
+    /// But for v5 transactions, this ID uniquely identifies the transaction's effects.
     #[allow(dead_code)]
-    pub fn effect_id(&self) -> Hash {
+    pub fn mined_id(&self) -> Hash {
         match self {
-            Narrow(effect_id) => *effect_id,
-            Wide(wtx_id) => wtx_id.id,
+            Legacy(effect_id) => *effect_id,
```

Comment:
Minor: Maybe this binding should be renamed as well?

```suggestion
            Legacy(legacy_id) => *legacy_id,
```

Reply:
Oops!